sys/kern/ringbuf.c

```diff
@@ -11,7 +11,8 @@ void ringbuf_init(ringbuf_t *rb, void *buf, size_t size) {
 }
 
 static void produce(ringbuf_t *buf, unsigned bytes) {
```

Comment:
Can't we just merge these two functions `produce` and `ringbuf_produce`? They do basically the same. Similar situation with `consume` and `ringbuf_consume`.

Reply:
I was thinking about it.  I did it that way because that assumption was weird to me in case of produce/consume function:
```
assert(buf->head + bytes <= buf->size);
...
if (buf->head == buf->size)
  buf->head = 0;
```
I deduced (perhaps wrongly) that there was any reasons for handle this function in that way therefore I did it next to that functions. Done - tests (ringbuf) passed.